librz/util/str.c

```diff
@@ -756,19 +756,11 @@ RZ_API char *rz_str_newf(const char *fmt, ...) {
 // Secure string copy with null terminator (like strlcpy or strscpy but ours
 RZ_API size_t rz_str_ncpy(char *dst, const char *src, size_t n) {
 	rz_return_val_if_fail(dst && src, 0);
-	size_t i;
 
-	// do not do anything if n is 0
-	if (n == 0) {
-		return 0;
-	}
+	memcpy(dst, src, n);
```

Comment:
This is not exactly the same as before, because before there was the check `src[i]`, which disappear if you use `memcpy`.

Reply:
i guess we could use `strncpy` instead of `memcpy` so it gets optimized by the compiler

Reply:
Good idea, we could remove everything and just keep `strncpy()` here.

Reply:
strncpy != strlcpy